sw/device/sca/aes_serial.c

```diff
@@ -37,10 +37,45 @@ enum {
    * samples).
    */
   kIbexAesSleepCycles = 400,
+  /**
+   * Max number of encryption that can be captured with the scope
+   * 81 is selected for AES with CW Husky
+   * Note: Maybe it would be better if we use dynamic memory allocation but I
+   * am not sure whether we are supporting it or not.
+   */
+  kNumBatchOpsMax = 81,
 };
 
+/**
+ * An array of keys to be used in a batch
+ */
+uint8_t batch_keys[kNumBatchOpsMax][kAesKeyLength];
+
+/**
+ * An array of plaintexts to be used in a batch
+ */
+uint8_t batch_plaintexts[kNumBatchOpsMax][kAesTextLength];
+
+/**
+ * Key selection between fixed and random key during the batch capture.
+ */
+bool sample_fixed = true;
+
+/**
+ * Fixed key for fvsr key TVLA batch capture
+ */
+uint8_t key_fixed[kAesKeyLength];
+
 static dif_aes_t aes;
 
+dif_aes_transaction_t transaction_shared = {
+    .operation = kDifAesOperationEncrypt,
+    .mode = kDifAesModeEcb,
+    .key_len = kDifAesKey128,
+    .masking = kDifAesMaskingInternalPrng,
+    .manual_operation = kDifAesManualOperationManual,
```

Comment:
Can you please also set the following settings? Some of these were only added recently and it's better to not rely on a default value as this may end up in painful debugging when the default changes.
- `key_provider` to `kDifAesSoftwareProvided`.
- `mask_reseeding` to `kDifAesReseedPerBlock`.
- `reseed_on_key_change` to `false` - this is highly important as we now change the key frequently. Meaning after every key change the AES unit is busy for a couple of cycles which disturbs the measurements.
- `reseed_on_key_change_lock` to `false`

Reply:
I guess you meant `kDifAesKeySoftwareProvided` instead of `kDifAesSoftwareProvided`, right?

Reply:
Sorry, yes you're absolutely right!